Description: Revert doctest changes for sympy 1.4
Bug: https://trac.sagemath.org/ticket/27653

Reverts the following commit:

From 3277ba76d0ba7174608a31a0c6623e9210c63e3d Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Fri, 12 Apr 2019 15:26:09 +0200
Subject: Adjust doctests for sympy 1.4 output syntax changes

---
 src/sage/calculus/calculus.py               |  2 +-
 src/sage/manifolds/chart_func.py            |  2 +-
 src/sage/manifolds/differentiable/metric.py |  4 ++--
 src/sage/manifolds/utilities.py             |  2 +-
 src/sage/typeset/ascii_art.py               | 10 +++++-----
 src/sage/typeset/character_art_factory.py   | 10 +++++-----
 src/sage/typeset/unicode_art.py             | 10 +++++-----
 7 files changed, 20 insertions(+), 20 deletions(-)

--- a/sage/src/sage/calculus/calculus.py
+++ b/sage/src/sage/calculus/calculus.py
@@ -1560,7 +1560,7 @@
     Testing SymPy::
 
         sage: laplace(t^n, t, s, algorithm='sympy')
-        (gamma(n + 1)/(s*s^n), 0, re(n) > -1)
+        (gamma(n + 1)/(s*s^n), 0, -re(n) < 1)
 
     Testing Maxima::
 
--- a/sage/src/sage/manifolds/chart_func.py
+++ b/sage/src/sage/manifolds/chart_func.py
@@ -2373,7 +2373,7 @@
             sage: f = X.function(sqrt(x^2-2*x+1)); f
             sqrt(x**2 - 2*x + 1)
             sage: f.simplify()
-            1 - x
+            -x + 1
 
         Power series expansion with respect to a small parameter `t` (at
         the moment, this is implemented only for the ``SR`` calculus backend,
--- a/sage/src/sage/manifolds/differentiable/metric.py
+++ b/sage/src/sage/manifolds/differentiable/metric.py
@@ -2302,8 +2302,8 @@
             sage: M.set_calculus_method('sympy')
             sage: g[1,1], g[1,2], g[2,2] = 1+x, x*y, 1-x
             sage: g[:]  # components in the manifold's default frame
-            [x + 1   x*y]
-            [  x*y 1 - x]
+            [ x + 1    x*y]
+            [   x*y -x + 1]
             sage: g.inverse()[:]
             [ (x - 1)/(x**2*y**2 + x**2 - 1)      x*y/(x**2*y**2 + x**2 - 1)]
             [     x*y/(x**2*y**2 + x**2 - 1) -(x + 1)/(x**2*y**2 + x**2 - 1)]
--- a/sage/src/sage/manifolds/utilities.py
+++ b/sage/src/sage/manifolds/utilities.py
@@ -776,7 +776,7 @@
 
         sage: s = (sqrt(x^2-2*x+1))._sympy_()
         sage: simplify_chain_real_sympy(s)
-        1 - x
+        -x + 1
 
     Other simplifications::
 
--- a/sage/src/sage/typeset/ascii_art.py
+++ b/sage/src/sage/typeset/ascii_art.py
@@ -266,11 +266,11 @@
 
         sage: n = var('n')
         sage: ascii_art(sum(binomial(2 * n, n + 1) * x^n, n, 0, oo))
-         /        _________    \
-        -\2*x + \/ 1 - 4*x  - 1/
-        -------------------------
-                   _________
-             2*x*\/ 1 - 4*x
+         /        __________    \
+        -\2*x + \/ -4*x + 1  - 1/
+        --------------------------
+                   __________
+             2*x*\/ -4*x + 1
         sage: ascii_art(list(DyckWords(3)))
         [                                   /\   ]
         [            /\    /\      /\/\    /  \  ]
--- a/sage/src/sage/typeset/character_art_factory.py
+++ b/sage/src/sage/typeset/character_art_factory.py
@@ -101,11 +101,11 @@
 
             sage: n = var('n')
             sage: ascii_art(sum(binomial(2 * n, n + 1) * x^n, n, 0, oo))
-             /        _________    \
-            -\2*x + \/ 1 - 4*x  - 1/
-            -------------------------
-                       _________
-                 2*x*\/ 1 - 4*x
+             /        __________    \
+            -\2*x + \/ -4*x + 1  - 1/
+            --------------------------
+                       __________
+                 2*x*\/ -4*x + 1
             sage: ascii_art(list(DyckWords(3)))
             [                                   /\   ]
             [            /\    /\      /\/\    /  \  ]
--- a/sage/src/sage/typeset/unicode_art.py
+++ b/sage/src/sage/typeset/unicode_art.py
@@ -132,11 +132,11 @@
 
         sage: n = var('n')
         sage: unicode_art(sum(binomial(2 * n, n + 1) * x^n, n, 0, oo))
-         ⎛        _________    ⎞
-        -⎝2⋅x + ╲╱ 1 - 4⋅x  - 1⎠
-        ─────────────────────────
-                   _________
-             2⋅x⋅╲╱ 1 - 4⋅x 
+             ⎛        __________    ⎞
+            -⎝2⋅x + ╲╱ -4⋅x + 1  - 1⎠
+            ──────────────────────────
+                       __________
+                 2⋅x⋅╲╱ -4⋅x + 1
         sage: unicode_art(list(DyckWords(3)))
         ⎡                                   ╱╲   ⎤
         ⎢            ╱╲    ╱╲      ╱╲╱╲    ╱  ╲  ⎥
